pdns/libssl.cc

```diff
@@ -806,6 +806,11 @@ std::unique_ptr<SSL_CTX, void(*)(SSL_CTX*)> libssl_init_server_context(const TLS
 {
   auto ctx = std::unique_ptr<SSL_CTX, void(*)(SSL_CTX*)>(SSL_CTX_new(SSLv23_server_method()), SSL_CTX_free);
```

Comment:
While we're at it, is it OK to use `decltype(&SSL_CTX_free)` here?

Reply:
Done!